_posts/en/newsletters/2021-11-03-newsletter.md

```diff
@@ -0,0 +1,180 @@
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/20-activation.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.2][] is a maintenance release for a previous branch
+  of Bitcoin Core [containing][bcc0.20.2 rn] minor features and
+  bug fixes.
+
+- [C-Lightning 0.10.2rc2][] is a release candidate that [includes][decker
+  tweet] a fix for the [uneconomical outputs security issue][news170
+  unec bug], a smaller database size, and an improvement in the
+  effectiveness of the `pay` command (see the *notable changes* section
+  below).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23306][] enables the Address Manager to support multiple ports per IP address.
+  Historically, Bitcoin Core has used a fixed port, 8333, and strongly preferred addresses with this
+  port when making automatic outbound connections. While it has been suggested that this behavior
+  may help prevent utilization of the Bitcoin nodes to DoS non-Bitcoin services (by gossipping their
+  addresses on the Bitcoin network), it also makes it easy to detect Bitcoin nodes by observing
+  network traffic. Treating each (IP,port) combination as a distinct address allows for future work
+  moving towards a more uniform treatment of addresses.
+
+- [C-Lightning #4837][] adds a `--max-dust-htlc-exposure-msat` configuration
+  option limiting the total balance of pending HTLCs whose amounts are below the
+  dust limit. Please see our [previous coverage][news162 mdhemsat] of a similar
+  option in Rust-Lightning for more details.
+
+- [Eclair #1982][] introduces a new log file collecting important notifications
+  that require the node operator to take action. The accompanying release notes
+  indicate that the `notifications.log` should be monitored by the node
+  operator.
+
+- [LND #5803][] allows multiple [spontaneous payments][topic spontaneous
+  payments] to the same [Atomic Multipath Payment (AMP)][topic multipath
+  payments] invoice without requiring the spending user perform extra
+  steps for repeat payments.  The ability to receive multiple payments
+  to the same invoice is a feature of AMP that's not possible with
+  existing [simplified multipath payment][topic multipath payments]
```

Comment:
It seems a little redundant to link to the same topic twice in the same paragraph:

```suggestion
  existing simplified multipath payment
```